docs/videos/public_agenda.md

```diff
@@ -0,0 +1,12 @@
+# All Aragon Devs #<number> (Agenda)
```

Comment:
Is this doc redundant with  docs/videos/template.md?

Reply:
It's used separately as the public agenda template vs. the notes template that is used during the call (and afterwards put up on the wiki).